src/zjs_ocf_ble.c

```diff
@@ -20,21 +20,10 @@ static ssize_t zjs_ble_storage_read(const bt_addr_le_t *addr, uint16_t key,
     return -EIO;
 }
 
-static int char2hex(const char *c, uint8_t *x)
-{
-    if (*c >= '0' && *c <= '9') {
-        *x = *c - '0';
-    } else if (*c >= 'a' && *c <= 'f') {
-        *x = *c - 'a' + 10;
-    } else if (*c >= 'A' && *c <= 'F') {
-        *x = *c - 'A' + 10;
-    } else {
-        return -EINVAL;
-    }
-
-    return 0;
-}
-
+/*
+ * Convert a device ID (MAC) string to  Zephyr bluetooth address
+ * e.g. input: "AA:BB:CC:DD:EE:FF"
+ */
 static int str2bt_addr_le(const char *str, const char *type, bt_addr_le_t *addr)
 {
     int i, j;
```

Comment:
`j` is no longer used. `int i` could go inside the for, doesn't really matter though.